policyengine_us/variables/gov/states/ms/tax/income/deductions/ms_deductions_indiv.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class ms_deductions_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = "Mississippi deductions filing seperately"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80108228.pdf#page=1"
+        "https://www.law.cornell.edu/regulations/mississippi/35-Miss-Code-R-SS-3-02-11-103"
```

Comment:
May add more source that directly explains an individual should claim itemized deductions or standard deduction, whichever is larger